core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -7608,9 +7608,9 @@ private void checkLiteral2(String expression, String expected) {
   }
 
   /** Test case for
-   * <a href="https://issues.apache.org/jira/browse/CALCITE-6156">[CALCITE-6156]
-   * Add ENDSWITH, STARTSWITH functions (enabled in Postgres, Snowflake libraries)</a>. */
-  @Test void testSnowflakeStartsWith() {
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6931">[CALCITE-6931]
```

Comment:
We can add Java documentation. It would be better not to delete the previous ones.You can refer to testHiveSparkAndBqTrim

Reply:
Thanks，done @NobiGo